app/gui/src/project-view/components/ComponentBrowser/input.ts

```diff
@@ -110,12 +111,20 @@ export function useComponentBrowserInput(
         : {}),
       }
     } else {
+      let literal: Ast.MutableTextLiteral | Ast.NumericLiteral | undefined =
+        Ast.TextLiteral.tryParse(text.value)
+      if (literal == null) {
+        literal = Ast.NumericLiteral.tryParse(text.value)
```

Comment:
There is a `tryParseWithSign` allowing negative numbers. This logic could use unit test coverage.

Reply:
The unit tests were once completely removed for `input.ts`, because CB got huge redesign and we didn't know if it will be stable. For now, I just (re) added tests for cases fixed by this PR.